api/libproject/devstart.rb

```diff
@@ -18,15 +19,20 @@ def initialize(command)
         self.project = project
       end
       parser.on("--account [ACCOUNT]",
-           "Account to use when creating credentials (your.name@pmi-ops.org)") do |account|
+           "Account to use when creating credentials (your.name@pmi-ops.org); "\
+           "use this or --creds_file") do |account|
         self.account = account
       end
+      parser.on("--creds_file [CREDS_FILE]",
+           "Path to a file containing credentials; use this or --account.") do |creds_file|
+        self.creds_file = creds_file
+      end
     end
   end
 
   def parse args
     self.parser.parse args
-    if self.project == nil || self.account == nil
+    if self.project == nil || !((self.account == nil) ^ (self.creds_file == nil))
```

Comment:
Another thing to note here: The `gsutil` command doesn't use the value of `--account` so it fails if the user's account is set to something that doesn't have access. Since that can't be changed for `gsutil` (as far as my own Googling says), we may want to use a different approach to avoid this problem.